src/System.Windows.Forms/src/System/Windows/Forms/ErrorProvider.ControlItem.ControlItemAccessibleObject.cs

```diff
@@ -31,8 +30,10 @@ public ControlItemAccessibleObject(ControlItem controlItem, ErrorWindow window,
 
                 internal override Rectangle BoundingRectangle => Bounds;
 
-                public override Rectangle Bounds => _control.IsHandleCreated ?
-                                                    _control.RectangleToScreen(_controlItem.GetIconBounds(_provider.Region.Size)) : Rectangle.Empty;
+                public override Rectangle Bounds =>
+                    _control.IsHandleCreated
```

Comment:
Please move `=>` to be consistent
```suggestion
                public override Rectangle Bounds
                    => _control.IsHandleCreated
```